raiden/ui/cli.py

```diff
@@ -204,23 +205,35 @@ def app(address,
 
         address = addresses[idx]
 
-    unlock_tries = 3
-    while True:
+    password = os.environ.get('ETH_PASSWORD')
```

Comment:
Humm, are we sure we want this? people could run raiden with:

`ETH_PASSWORD=password raiden ...`

And I believe this would end up in the shell history.

The prompt was used exactly to avoid this sort of problem, required the user to type the password on a no-echo terminal.

@ulope @LefterisJP @konradkonrad ?